client/src/store/modules/groups.js

```diff
@@ -8,7 +8,13 @@ const state = {
 // getters
 const getters = {
 
-  groupsAll: state => state.all
+  groupsAll: state => state.all,
+
+  groupNameById (state) {
```

Comment:
Would be better to get group object from the Store, not only `name` field.  Another words `groupById`